mssql/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for mssql v3.1.0
+// Type definitions for mssql v3.3.0
```

Comment:
version should be `major.minor` only, and no `v` prefix, i.e. `// Type definitions for mssql 3.3`